andela/ah-frontend-thanos
# #161348755 User should be able to login

#### What does this PR do?

- The user should be able to log in through the login form

#### Description of Task to be completed?

- A user should be able to login.
- User data should be validated.
- Messages after successful login and error messages if login is invalid
- Should be able to redirect to the sign-up page

#### How should this be manually tested?

- Go to:
````
Url: /login
````
- Provide a valid email and password in the form(If a signed up user)
- It should then be redirected to the home page.

#### What are the relevant pivotal tracker stories?

[#161348755](https://www.pivotaltracker.com/story/show/161348755)

#### Screenshots 

##### Nav bar
![image](https://user-images.githubusercontent.com/39259834/49380746-65490380-f723-11e8-811c-75cf283bdc0a.png)

##### Login form
![image](https://user-images.githubusercontent.com/39259834/49380793-8578c280-f723-11e8-8d0a-0ce1788a474c.png)

##### Footer
![image](https://user-images.githubusercontent.com/39259834/49380809-91fd1b00-f723-11e8-8b22-0be5074834e0.png)

##### Whole page
![image](https://user-images.githubusercontent.com/39259834/49380829-9de8dd00-f723-11e8-8782-d043c86b0add.png)
